### DISCUSSION TITLE
Ability to restrict home menu videos to subscriptions only

### DISCUSSION THREAD

**Discussion by DraconicNEO** [ReVanced/revanced-discussions#685] [Question]
I think it will be an interesting idea for a patch that would restrict the videos that can appear on your home screen to videos from subscriptions only, the way that it was originally on YouTube before the recommendations system. 

The reason is that I know that some people find the random recommendations appearing on their home screen to be bothersome if they only want to see videos from their subscriptions.

**Comment by LisoUseInAIKyrios** [comment]
You can now hide navigation tabs

**Reply by DraconicNEO** [reply]
Do other recommended videos outside of channels I subscribed to still appear when doing that?

**Reply by LisoUseInAIKyrios** [reply]
You will still see recommended videos under the video player, and that includes channels you do not subscribe to.  But the home tab itself would be gone, and only the subscription tab and library would be left.

**Reply by DraconicNEO** [reply]
Yeah so that wasn't really what I had in mind, was thinking more along the lines of something that only includes videos from subscribed channels in the home menu's grid, as opposed to just hiding it in favor of the subscribed list.

**Reply by LisoUseInAIKyrios** [reply]
Because YT pushes the recommendations, the most ReVanced can do it selectively hide stuff.  It has no control over what is pushed to the client.
